docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -255,7 +255,39 @@ Initialize bond and token asset states and ownerships on the `network1` ledger b
 ```bash
 ./scripts/initAssetsForTransfer.sh
 ```
+The `initAssetsForTranser.sh` script above also fetches the certificate of the users in base64 format and stores in the folder `src/data/remoteNetworkUsers` with filename `network1_UsersAndCerts.json` and `network2_UsersAndCerts.json` corresponding to `network1` and `network2` respectively. These files are referred during pledge to fetch the certificate of the recipient of the transferred asset, and during claim to fetch the certificate of the pledger of the asset. *This is because, asset transfer doesn't require both the users being members of neither in the export nor in the import network. It's just sufficient that pledger is part of the asset export network and claimer is part of the asset import network*.
+  * The user certificate files corresponding to the Fabric networks also need to be copied into this folder.
```

Comment:
I think you mean `Corda networks` here? Because by design of fabric-cli, for fabric networks they will be already generated here.

Reply:
Yes, Sandeep. Updated now.